src/devices/CharacterLcd/samples/PCF8574T Sample/PCF8574T I2C LCD Sample.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00

```

Comment:
I'd prefer if you've replaced all spaces in file and directory names with dashes

Reply:
Apart from removing spaces on path names, can we also just remove the solution?